script/bootstrap.py

```diff
@@ -88,8 +89,14 @@ def parse_args():
   parser.add_argument('--clang_dir', default='', help='Path to clang binaries')
   parser.add_argument('--disable_clang', action='store_true',
                       help='Use compilers other than clang for building')
-  parser.add_argument('--build_libchromiumcontent', action='store_true',
```

Comment:
Should be updated now. Also removed reference to the explicit copy/usage of `.gclient` in build-instructions-linux.md since that file is now automatically generated. Also note that build-libchromiumcontent.py is now executed in the libchromiumcontent directory.

Hoping that this change was OK even though it removes the flexibility of manually configuring .gclient for chromium code checkout.